scripts/docs/bin/build_docs.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
I know it makes everyone but my life harder to use the OS provided python3, but I want it to stay that way - I don't want to have to try and make sure that the same python environment is being used by makefiles, pip installs, local builds, CI builds, release scripts, etc.

Reply:
Ah, yes, you can revert this when merging. I'm still on Mojave, so I needed to make the change to make Hammerspoon build.